### PR TITLE
Added marker icon if picture is empty on SfStore component.

### LINKED ISSUE #1280: [FEATURE] SfSToreLocator: add placeholder for empty picture
If the store data does not have any picture, then a black space is shown.

We should remove the black space and add a placeholder when the picture is missing.

### PR DESCRIPTION
**Pull request by mohammad-k8**
# Related issue

Closes #1280 

# Scope of work

# Screenshots of visual changes
Screenshot 2020-07-26 at 2 02 18 PM

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))
 - [ ] I accept [Individual Contributor License Agreement](https://github.com/DivanteLtd/next/blob/master/CLA.md)

### CODE REVIEW

File: packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue [hunk 1]
```diff
@@ -6,15 +6,17 @@
         <span>{{ distance }}km</span> away from you
       </div>
     </slot>
-    <div class="sf-store__media" @click="centerOn(latlng)">
+    <div v-bind:class="{ 'sf-store__media': picture }" @click="centerOn(latlng)">
```
Comment (Michal-Dziedzinski) [inline packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue:9]:
```suggestion
    <div :class="{ 'sf-store__media': picture }" @click="centerOn(latlng)">
```

Wdyt about using shorthand?

File: packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue [hunk 2]
```diff
@@ -6,15 +6,17 @@
...
       <!-- @slot Use this slot to show media elements -->
       <slot name="media">
         <SfImage
+          v-if="picture"
           :src="picture"
           :alt="`${name} picture`"
           :width="82"
           :height="112"
         />
+        <SfIcon v-if="!picture" icon="marker" />
```
Comment (Michal-Dziedzinski) [inline packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue:19]:
```suggestion
        <SfIcon v-else icon="marker" />
```

Is `v-else` better in that place. isn't it?

File: packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue (hunk 1 above)
Comment (mohammad-k8) [reply packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue:9]: @Michal-Dziedzinski Thanks for the valid suggestion. I'll update the PR according to it.

File: packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue (hunk 2 above)
Comment (mohammad-k8) [reply packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue:19]: @Michal-Dziedzinski Thanks for the valid suggestion. I'll update the PR according to it.

**Comment by AdamPawlinski** [comment]
Work continued on https://github.com/DivanteLtd/storefront-ui/pull/1387

**Pull request closed without merging**